src/highrise/models.py

```diff
@@ -11,6 +11,8 @@ class User:
 
 Reaction: TypeAlias = Literal["clap", "heart", "thumbs", "wave", "wink"]
 Facing: TypeAlias = Literal["FrontRight", "FrontLeft", "BackRight", "BackLeft"]
+RoomPermission: TypeAlias = Literal["moderator", "designer"]
+ModerationAction: TypeAlias = Literal["kick", "ban", "unban", "mute"]
```

Comment:
Hm this literal is used only once, can we inline it into the request?

I think it might produce clearer autocomplete for users in Discord?

Reply:
Fixed